RefApp/ViewControllers/EnvironmentViewController.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  EnvironmentViewController.swift
+//  RefApp
+//
+//  Created by Udaya Sri Senarathne on 2018-11-21.
+//  Copyright © 2018 emp. All rights reserved.
+//
+
+import UIKit
+import Exposure
+
+class EnvironmentViewController: UIViewController {
+    
+    let topHolderView: UIView = {
+        let view = UIView()
+        return view
+    }()
+    
+    let bottomHolderView: UIView = {
+        let view = UIView()
+        return view
+    }()
+    
+    let logoImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "redbee")
+        imageView.contentMode = .scaleAspectFill
+        return imageView
+    }()
+    
+    let environmentUrlTextField: RBMTextField = {
+        let textfield = RBMTextField(placeHolderText: "Exp URL")
```

Comment:
Nothing major but maybe a better placeholder that explains the url:

"Exposure base URL"?

Reply:
Yes, let changed it to Exposure base URL